### PR TITLE
Fix array descriptions

### PR DESCRIPTION
**Pull request by joncol**
Instead of saying "The value of this array must match *at least one* of the following schemas", the description will say "Each element of this array must match *at least one* of the following schemas".

### CODE REVIEW

File: json-schema-to-html/schema.js [hunk 1]
```diff
@@ -130,19 +130,19 @@ function renderItemsValidation(schema, type, octothorpes) {
 	var text = []
 	var validationItems = []
 
-	var content = schema.type == "array" ? "elements" : "value";
+	var content = schema.type == "array" ? "Each element" : "The value";
```
Comment (zlondrej) [inline json-schema-to-html/schema.js]: The comparison has to be against `type` and not `schema.type`.

**Review by zlondrej — CHANGES_REQUESTED** [review]

File: json-schema-to-html/schema.js (hunk 1 above)
Comment (joncol) [reply json-schema-to-html/schema.js]: Ooops, did the change locally, but failed to commit it :face_in_clouds:.

**Comment by zlondrej** [comment]
Your tests are failing.

**Review by zlondrej — APPROVED** [review]

**Comment by joncol** [comment]
@zlondrej: I added a changelog entry for this as well. Will create the release in a separate PR, after merging the other (Nix flake) PR.

**Review by zlondrej — APPROVED** [review]

**Pull request merged**